packages/base-styles/_z-index.scss

```diff
@@ -71,9 +71,6 @@ $z-layers: (
 	".edit-site-sidebar": 100000,
 	".edit-widgets-sidebar": 100000,
 	".edit-post-layout .edit-post-post-publish-panel": 100001,
-	// For larger views, the wp-admin navbar dropdown should be at top of
```

Comment:
What was the reason for removing this style? It seems unrelated.

Reply:
i believe this z-index is not right when you have multiple sidebars, but I'll restore it for now.